sw/device/tests/penetrationtests/firmware/sca_lib.c

```diff
@@ -21,6 +27,112 @@ static dif_rv_plic_t plic;
 static dif_rstmgr_t rstmgr;
 static dif_alert_handler_t alert_handler;
 
+status_t sca_configure_entropy_source_otbn(void) {
```

Comment:
nit: I'd drop the `_otbn`, because this routine will configure EDN for all blocks, not just OTBN. Maybe just `sca_configure_entropy_source`, or `sca_configure_entropy_source_max_reseed_interval`?

Reply:
Thanks, good point! I've renamed the function to `sca_configure_entropy_source_max_reseed_interval`